### ISSUE TITLE
Could not locate `cudnn_ops_infer64_8.dll`. Please make sure it is in your library path!

### ISSUE THREAD

**Issue by YoungPhlo** [m-bain/whisperX#899]
Command:
```pwsh
whisperx "audio/test.mp3" --model "large-v3" --language "en"
```
Error:
```pwsh
Could not locate cudnn_ops_infer64_8.dll. Please make sure it is in your library path!
```

Took a look in `.venv\Lib\site-packages\torch\lib` and saw `cudnn_ops_infer64_9.dll` instead of `cudnn_ops_infer64_8.dll`

I originally used these commands to set up `whisperX` today:
```pwsh
py -3.10 -m venv venv
.\venv\Scripts\python.exe -m pip install --upgrade pip wheel setuptools
pip install torch torchvision torchaudio --index-url https://download.pytorch.org/whl/cu121
pip install git+https://github.com/m-bain/whisperx.git
```

After seeing the error I ran:
```pwsh
pip uninstall torch torchaudio torchvision
pip install torch==2.2.2 torchvision==0.17.2 torchaudio==2.2.2 --index-url https://download.pytorch.org/whl/cu121
```

and `cudnn_ops_infer64_8.dll` is in torch so the original command runs now

```
- Operating System
OS Name:         Microsoft Windows 10 Pro
Version:         10.0.19045
Build:           19045

- Environment
PowerShell:      5.1.19041.5007
Python:          3.10.11
Git:             2.41.0.windows.1
Node.js:         20.15.1
```

**Comment by DaddyCodesAlot** [comment]
It could be the new ctranslate2 update that bonked things. Downgrading ctranslate2 fixed this issue for me.

I just used this command:
pip install ctranslate2==4.4.0

**Comment by Leandrocnf** [comment]
> It could be the new ctranslate2 update that bonked things. Downgrading ctranslate2 fixed this issue for me.
> 
> I just used this command: pip install ctranslate2==4.4.0

I had a different issue, but it was solved using this same command. Thank you so much!

**Comment by pintoseb** [comment]
> It could be the new ctranslate2 update that bonked things. Downgrading ctranslate2 fixed this issue for me.
> 
> I just used this command: pip install ctranslate2==4.4.0

Worked for me also. Thank you so much.

**Comment by Barabazs** [comment]
Fixed in latest release: [v3.2.0](https://github.com/m-bain/whisperX/releases/tag/v3.2.0)

**Comment by YoungPhlo** [comment]
> Fixed in latest release: [v3.2.0](https://github.com/m-bain/whisperX/releases/tag/v3.2.0)

Just wanted to say thank you for this new release, amazing job 🔥

**Comment by vitaliy-sharandin** [comment]
Simplest solution - use Python 3.10/3.11 with 2.1.2+cu121 torch, as it already has that lib.

**Comment by Adheesh1107** [comment]
@mlallan1307 , Can you please mention which version of python and WhisperX you were using?

**Comment by BestofthebestinAI** [comment]
And show your full pip list?

**Comment by Brodski** [comment]
> Even on v3.2.0 release and ensuring ctranslate2==4.4.0 I still had this issue. This is what fixed it for me [SYSTRAN/faster-whisper#715 (comment)](https://github.com/SYSTRAN/faster-whisper/discussions/715#discussioncomment-10903540)
> 
> Download cuDNN v8.9.7 for cuda 12.x https://developer.nvidia.com/rdp/cudnn-archive
> 
> `pip show ctranslate2`
> 
> Open the folder location of ctranslate2 which has a file "cudnn64_8.dll" in it
> 
> From the cuDNN zip file, open bin/ and copy all files into the ctranslate2 folder

Thanks! I was able to download it w/o an account at cudnn-downloads archive: https://developer.download.nvidia.com/compute/cudnn/redist/cudnn/

Also, youre still having issues, you might need to update your env variables. I added `C:\Program Files\NVIDIA\CUDNN\v8.9.7_cuda12\bin` to the Path var, under System Variables
